trainingMaquetado/Maquetado.css

```diff
@@ -0,0 +1,49 @@
+hr {
```

Comment:
target classes instead of html elements, although this isn't wrong we tend to style classes because they are the link between structure (HTML) and styles (CSS). To learn more about this please read this document (https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style.guide.md)